packages/api/src/module/write/email-confirmation/domain/requestEmailConfirmation.ts

```diff
@@ -0,0 +1,9 @@
+import { RequestEmailConfirmation } from '@/module/commands/request-email-conformation';
+import { emailConfirmationWasRequestedEvent } from '@/module/events/email-confirmation-was-requested.domain-event';
+
+import { RequestEmailConfirmationDomainEvent } from './events';
+
+export const requestEmailConfirmation =
```

Comment:
I'm not sure about this construct, that function return function. It is readable for you?

Reply:
Ah, `requestEmailConfirmation(command)` will look nice :D Leave it as you have done.